ansible/install_td_agent.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: win
+  gather_facts: false
+  tasks:
+    - name: Ping test
+      win_ping:
+- hosts: localhost
+  vars:
+    td_agent_file: "td-agent-3.5.0-0-x64.msi"
+    td_agent_installer: "http://packages.treasuredata.com.s3.amazonaws.com/3/windows"
+    td_agent_fetch_url: "{{ td_agent_installer }}/{{ td_agent_file }}"
+  tasks:
+    - name: Create temporary target directory if not exists
+      tempfile:
+        state: directory
+      register: tempdir
+    - name: Fetch td-agent msi
+      get_url:
+        url: "{{ td_agent_fetch_url }}"
+        dest: "{{ tempdir.path }}/td-agent.msi"
+- hosts: win
+  tasks:
+    - name: Create temporary target directory
+      win_tempfile:
+        state: directory
+      register: tempdir_win
+    - name: Copy td-agent msi
+      win_copy:
+        src: "{{ hostvars['localhost']['tempdir']['path'] }}/td-agent.msi"
+        dest: "{{ tempdir_win.path }}\\"
+    - name: Run td-agent msi
```

Comment:
I see the `when` clause being used in #21. Shouldn't that be used here too? I would like to have consistency regarding such things across our playbooks.

Reply:
As a part of squash, this yaml file would be dropped as it is not needed